docs/content/contributing/docs/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+type: docs
+title: "Radius Docs"
+linkTitle: "Docs"
+description: "Learn how to contribute and maintain the Radius documentation"
```

Comment:
```suggestion
description: "Learn how to maintain the Radius documentation"
```

Reply:
This is the the root index page that has sub pages on both contributing and maintaining.